### PR TITLE
:zap: Added static pages

### PR DESCRIPTION
**Pull request by ajcpwnz**
Added views for static pages. We will remove this views, but can use controller to serve more static pages in future. 
Also set up some SASS asset structure

### CODE REVIEW

File: app/assets/stylesheets/_reset.sass [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  opacity: 1
+input:-moz-placeholder
+  color: inherit
+  opacity: 1
+::-moz-placeholder
+  color: inherit
+  opacity: 1
+input:-ms-input-placeholder
+  color: inherit
+  opacity: 1
```
Comment (IlyaUmanets) [inline app/assets/stylesheets/_reset.sass]: Add new line.

File: app/assets/stylesheets/application.sass [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@import 'reset'
+@import 'variables'
+@import 'bem/**/*'
```
Comment (IlyaUmanets) [inline app/assets/stylesheets/application.sass]: Same

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class PagesController < ApplicationController
+  def show
+    layout = case params[:page]
```
Comment (IlyaUmanets) [inline app/controllers/pages_controller.rb]: Move this to private method `layout_name`.
And use `a ? b : c` approach

Comment (ajcpwnz) [inline app/controllers/pages_controller.rb]: I didn`t use ternary operator because in future we can have more than two layouts

**Pull request closed without merging**